Calibration.py

```diff
@@ -153,35 +142,22 @@ def read_config(self):
 
             # Read the motor min/mid/max servo ranges
             if self.core is not None:
-                # Read Wheel motor ESC ranges
-                self.core.left_servo.servo_min =\
-                    int(config.get('motors', 'LEFT_MIN'))
-                self.core.left_servo.servo_mid =\
-                    int(config.get('motors', 'LEFT_MID'))
-                self.core.left_servo.servo_max =\
-                    int(config.get('motors', 'LEFT_MAX'))
-
-                self.core.right_servo.servo_min =\
-                    int(config.get('motors', 'RIGHT_MIN'))
-                self.core.right_servo.servo_mid =\
-                    int(config.get('motors', 'RIGHT_MID'))
-                self.core.right_servo.servo_max =\
-                    int(config.get('motors', 'RIGHT_MAX'))
-
-                # Read Auxilery ESC ranges
-                self.core.left_aux_1_servo.servo_min = \
-                    int(config.get('motors', 'LEFT_AUX_1_MIN'))
-                self.core.left_aux_1_servo.servo_mid = \
-                    int(config.get('motors', 'LEFT_AUX_1_MID'))
-                self.core.left_aux_1_servo.servo_max = \
-                    int(config.get('motors', 'LEFT_AUX_1_MAX'))
-
-                self.core.right_aux_1_servo.servo_min = \
-                    int(config.get('motors', 'RIGHT_AUX_1_MIN'))
-                self.core.right_aux_1_servo.servo_mid = \
-                    int(config.get('motors', 'RIGHT_AUX_1_MID'))
-                self.core.right_aux_1_servo.servo_max = \
-                    int(config.get('motors', 'RIGHT_AUX_1_MAX'))
+                # Read in min/mid/max of menu items that we can change.
+                for item in self.menu_list.items():
+                    item_enum_str = str(item[0]).split('.')
+                    item_min_str = "{}_MIN".format(item_enum_str[0])
+                    item_mid_str = "{}_MID".format(item_enum_str[0])
+                    item_max_str = "{}_MAX".format(item_enum_str[0])
+                    try:
+                        self.core.servos[item[0]][0].servo_min = \
```

Comment:
That was my thought at midnight. Didn't like it, but its tricky to see clearly how the dict inside dict would be accessed.